_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<p>Define the fields in this table.  The testing bias ratio is not defined in this table. </p><p> </p><p>For example see "Definition of Fields" in this example: https://covid19dashboards.com/covid-19-us-case-estimation/#Definition-Of-Fields.&nbsp;Maybe you are already doing this but it is unclear which definitions go with which field.&nbsp;Make this explicit.&nbsp;</p><p> </p><p>This chart isn't that valuable without ICU capacity, and the comparison is not that useful, is there a way to include capacity in here so we know which countries are in trouble?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I don't have per countries capacities for most countries. I've linked to what information I have more prominantly at the top of that section instead of only in appendix.</p><p> </p><p>Given the narrow range of ICU capacities in the link I've added, I think the comparative numbers (between countries, and between time ranges) are quite useful due to the differences in magnitude. </p><p> </p><p> </p><p>From just observing the past data - it seems that countries are in trouble earlier than their capacity is reached (probably because the trouble is localized), so to me it seems that any number above 1 is already in crisis mode (at least from countries that got into that territory (e.g. Italy).</p><p> </p><p>Of course, It shouldn't be difficult to add the capacity later if a data source is found (or if someone who has better ideas finds this). </p>